package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@prisma/language-support",
-  "version": "7.0.3",
+  "version": "1.0.0",
```

Comment:
Don't we want to keep this in sync still?

Reply:
yes, I'll change that

Reply:
not done yet - not pushed?